packages/batch-submitter/src/types.ts

```diff
@@ -1,23 +1,12 @@
 /* External Imports */
 import {
   BlockWithTransactions,
-  Provider,
   TransactionResponse,
 } from '@ethersproject/abstract-provider'
 
-/* Internal Imports */
-import { EIP155TxData, TxType } from '@eth-optimism/core-utils'
-
 export enum QueueOrigin {
```

Comment:
Potentially out of scope of this PR but this enum should be moved to `core-utils`

Reply:
I notice there is another change to `core-utils` as part of this PR